frontend/src/pages/authorizationDetails/ComplianceSection.tsx

```diff
@@ -0,0 +1,143 @@
+import { Stack, Typography } from '@mui/material'
+import ArrowUpwardIcon from '@mui/icons-material/ArrowUpward'
+import ArrowDownwardIcon from '@mui/icons-material/ArrowDownward'
+import ImportExportIcon from '@mui/icons-material/ImportExport'
+import OpenInNewIcon from '@mui/icons-material/OpenInNew'
+import { useEffect, useState } from 'react'
+import './ComplianceSection.css'
+import OmrrData from '@/interfaces/omrr'
+
+interface Props {
+  item: OmrrData
+}
+
+interface ComplianceRecord {
+  dateIssued: string
+  recordType: string
+  summary: string
+  id: string
+}
+
+export function ComplianceSection({ item }: Readonly<Props>) {
+  const [complianceData, setComplianceData] = useState<ComplianceRecord[]>([])
+  const [sortDirection, setSortDirection] = useState<'asc' | 'desc' | null>(
+    null,
+  )
+
+  useEffect(() => {
+    const fetchComplianceData = async () => {
+      const keywords = `${item['Authorization Number']} OR "${item['Regulated Party']}"`
+      const url = `https://nrpti-api-f00029-prod.apps.silver.devops.gov.bc.ca/api/public/search?dataset=OrderNRCED,InspectionNRCED,RestorativeJusticeNRCED,AdministrativePenaltyNRCED,AdministrativeSanctionNRCED,TicketNRCED,WarningNRCED,CourtConvictionNRCED,CorrespondenceNRCED,DamSafetyInspectionNRCED,ReportNRCED&keywords=${encodeURIComponent(keywords)}&pageNum=0&pageSize=25&sortBy=-dateIssued&fields=`
```

Comment:
lets add these base urls as env vars to frontend, for easy future maintainance
`https://nrpti-api-f00029-prod.apps.silver.devops.gov.bc.ca/`